custom-vu/src/main/kotlin/jces1209/vu/action/ViewBoard.kt

```diff
@@ -49,16 +52,31 @@ class ViewBoard(
             }
         )
 
+
         if (random.random.nextFloat() < viewIssueProbability) {
             if (boardContent.getIssueKeys().isEmpty()) {
                 logger.debug("It requires some issues on board to test preview issue")
             } else {
+                val jiraTips = JiraTips(driver)
                 jiraTips.closeTips()
                 meter.measure(MeasureType.ISSUE_PREVIEW_BOARD) {
                     meter.measure(ActionType("Preview issue ($boardType board)") { Unit }) {
                         board.previewIssue()
                     }
                 }
+                board.closePreviewIssue()

```

Comment:
Just a code-style nit. I can no longer fit the entire method on a single screen.